src/index.html

```diff
@@ -10,8 +10,173 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>HTML Form</h1>
+  <body class="body">
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <div class="container top-container">
+          <label class="label" for="surname">Surname:
+            <input type="text"
+                   name="surname"
+                   autocomplete="off"
+                   id="surname"
+                   required>
```

Comment:
```suggestion
          <label class="label" for="surname">
            Surname:
            <input
              type="text"
              name="surname"
              autocomplete="off"
              id="surname"
              required
            >
```
Зверни увагу на відступи і форматування і виправ скрізь.
Про це можна почитати тут
https://github.com/mate-academy/style-guides/blob/master/htmlcss.md#indentation